contracts/deposit-contract/depositContract.go

```diff
@@ -214,30 +214,30 @@ func (_DepositContract *DepositContractCallerSession) CHAINSTARTFULLDEPOSITTHRES
 	return _DepositContract.Contract.CHAINSTARTFULLDEPOSITTHRESHOLD(&_DepositContract.CallOpts)
 }
 
-// MAXDEPOSITAMOUNT is a free data retrieval call binding the contract method 0x4c34a982.
+// FULLDEPOSITAMOUNT is a free data retrieval call binding the contract method 0x98faf4ba.
 //
-// Solidity: function MAX_DEPOSIT_AMOUNT() constant returns(uint256 out)
-func (_DepositContract *DepositContractCaller) MAXDEPOSITAMOUNT(opts *bind.CallOpts) (*big.Int, error) {
+// Solidity: function FULL_DEPOSIT_AMOUNT() constant returns(uint256 out)
+func (_DepositContract *DepositContractCaller) FULLDEPOSITAMOUNT(opts *bind.CallOpts) (*big.Int, error) {
 	var (
 		ret0 = new(*big.Int)
 	)
 	out := ret0
-	err := _DepositContract.contract.Call(opts, out, "MAX_DEPOSIT_AMOUNT")
+	err := _DepositContract.contract.Call(opts, out, "FULL_DEPOSIT_AMOUNT")
 	return *ret0, err
 }
 
-// MAXDEPOSITAMOUNT is a free data retrieval call binding the contract method 0x4c34a982.
+// FULLDEPOSITAMOUNT is a free data retrieval call binding the contract method 0x98faf4ba.
 //
-// Solidity: function MAX_DEPOSIT_AMOUNT() constant returns(uint256 out)
-func (_DepositContract *DepositContractSession) MAXDEPOSITAMOUNT() (*big.Int, error) {
-	return _DepositContract.Contract.MAXDEPOSITAMOUNT(&_DepositContract.CallOpts)
+// Solidity: function FULL_DEPOSIT_AMOUNT() constant returns(uint256 out)
+func (_DepositContract *DepositContractSession) FULLDEPOSITAMOUNT() (*big.Int, error) {
+	return _DepositContract.Contract.FULLDEPOSITAMOUNT(&_DepositContract.CallOpts)
 }
 
-// MAXDEPOSITAMOUNT is a free data retrieval call binding the contract method 0x4c34a982.
+// FULLDEPOSITAMOUNT is a free data retrieval call binding the contract method 0x98faf4ba.
 //
-// Solidity: function MAX_DEPOSIT_AMOUNT() constant returns(uint256 out)
-func (_DepositContract *DepositContractCallerSession) MAXDEPOSITAMOUNT() (*big.Int, error) {
-	return _DepositContract.Contract.MAXDEPOSITAMOUNT(&_DepositContract.CallOpts)
+// Solidity: function FULL_DEPOSIT_AMOUNT() constant returns(uint256 out)
+func (_DepositContract *DepositContractCallerSession) FULLDEPOSITAMOUNT() (*big.Int, error) {
```

Comment:
are we sure we want to have our function names capitalized like that

Reply:
@shayzluf this is generated code

Reply:
gotcha